### ISSUE TITLE
Bulk upload data for new dataproject from AnnoRep

### ISSUE THREAD

**Issue by nniiicc** [QualitativeDataRepository/AnnoREP-Frontend#41]
## Use Case

As a user of AnnoRep - I want to bulk upload my data when creating a new dataproject 

## Solution

This is related to issue #40 
After a user creates a dataproject, it would be valuable to let them immediately add any data they would like to use in annotations to that new project. Once a project is created - the manuscript screen should display all of the data sources that have been uploaded (in the left hand column)

## Alternatives

_Please describe any alternatives you've considered, even if you've dismissed them_

**Comment by tohuynh** [comment]
@adam3smith Is this "on hold" too? We are duplicating QDR's functionality here. Users can bulk upload data files when they create a new data project.

**Comment by adam3smith** [comment]
Yeah. This one may depend a bit more on user feedback and usage patterns. If people frequently want to add individual files while annotating, this may be worth it. If they're mostly uploading files in bulk, the QDR interface is perfectly fine
